Cargo.toml

```
@@ -18,3 +18,7 @@ path = "src/lib.rs"
 [dependencies]
 clap = "2.33.0"                                                         # https://github.com/clap-rs/clap
 pulldown-cmark = { version = "0.5", default-features = false }          # https://github.com/raphlinus/pulldown-cmark
+
+[dev-dependencies]
+assert_cmd = "0.11.1"
```

I haven't used these packages before but they look useful! I have a habit of linking to the github repo though, just for convenience :)

```suggestion
assert_cmd = "0.11.1"                                                   # https://github.com/assert-rs/assert_cmd
```